app/models/user.rb

```diff
@@ -61,7 +61,35 @@ def pciu_alternate_phone
   end
 
   def first_name
-    identity.first_name || (mhv_icn.present? ? mpi&.profile&.given_names&.first : nil)
+    identity.first_name || (mhv_icn.present? ? va_profile&.given_names&.first : nil)
+  end
+
+  def given_names
+    mpi.profile&.given_names
+  end
+
+  def suffix
```

Comment:
This is a huge bugbear of mine, I agree with removing the safe operators in User. In fact, I put a story in the backlog to address this issue. I don't think this specific PR should cover it, but our data consistency would definitely improve if we dealt with this data inconsistency in a sensible manner. I think we should have a more general conversation about it anyway, because we should probably figure out standards of logging when there is missing or inconsistent data, instead of the equivalent of `rescue nil` all over the place